lib/spack/spack/test/hg_fetch.py

```diff
@@ -21,6 +22,8 @@
     not which('hg'), reason='requires mercurial to be installed')
 
 
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason="Not supported on Windows (yet)")
```

Comment:
IMO if the only tests that work in a module involve testing for exceptions/failures/noops, then we should disable the whole module: in this case `test_hg_extra_fetch` may pass, but I think that's more misleading than helpful.